### PR TITLE
Add preview media loader, disable default media loader in preview mode

### PR DESCRIPTION
**Pull request by ahilles107**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | yes
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | yes
| Fixed tickets             | SWP-825
| License                   | AGPLv3

TODO:

* [ ] tests
* [ ] update changelog

### CODE REVIEW

File: src/SWP/Bundle/ContentBundle/Factory/ORM/MediaFactory.php [hunk 1]
```diff
@@ -116,6 +101,44 @@ public function createImageRendition(
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function replaceBodyImagesWithMedia(ArticleInterface $article, ArticleMediaInterface $articleMedia)
```
Comment (takeit) [inline src/SWP/Bundle/ContentBundle/Factory/ORM/MediaFactory.php]: why is this method in the factory? it should be in some different service (ArticleBodyProcessor etc.) It doesn't fit here IMO.

File: src/SWP/Bundle/CoreBundle/Resources/config/services.yml [hunk 2]
```diff
@@ -279,3 +279,13 @@ services:
...
+
+    swp_core.loader.media:
+        class: SWP\Bundle\CoreBundle\Loader\PreviewArticleMediaLoader
+        arguments:
+            - "@swp.factory.media"
+            - "@swp_template_engine_context.factory.meta_factory"
+            - "@swp_template_engine_context"
+            - "@request_stack"
+        tags:
+            - { name: swp.meta_loader.add }
```
Comment (takeit) [inline src/SWP/Bundle/CoreBundle/Resources/config/services.yml]: new line is missing

File: src/SWP/Bundle/TemplatesSystemBundle/DependencyInjection/ContainerBuilder/MetaLoaderCompilerPass.php [hunk 3]
```diff
@@ -31,13 +31,23 @@ public function process(ContainerBuilder $container)
...
         );
         foreach ($taggedServices as $id => $tags) {
             $definition->addMethodCall(
                 'addLoader',
                 [new Reference($id)]
             );
         }
+
+        $taggedServices = $container->findTaggedServiceIds(
+            'swp.meta_loader.remove'
```
Comment (takeit) [inline src/SWP/Bundle/TemplatesSystemBundle/DependencyInjection/ContainerBuilder/MetaLoaderCompilerPass.php]: it looks that there are no services registered with this tag, what is it used for?

Comment (ahilles107) [inline src/SWP/Bundle/TemplatesSystemBundle/DependencyInjection/ContainerBuilder/MetaLoaderCompilerPass.php]: it a way to de register in future content loader (if you want to provide your own version of it).

Comment (takeit) [inline src/SWP/Bundle/TemplatesSystemBundle/DependencyInjection/ContainerBuilder/MetaLoaderCompilerPass.php]: then you can do this by, for example, writing compiler pass and override the class for the definition

**Pull request merged**